scripts/github-actions/upload-modules-metadata.js

```diff
@@ -1,7 +1,7 @@
 /**
  * @param {typeof import("@actions/core")} core
  */
-async function uploadModulesMetadata(core) {
+async function uploadModulesMetadata({ require, core }) {
```

Comment:
Can you also update the JSDoc comment? Here's an example: https://github.com/Azure/bicep-registry-modules/blob/917735e53c4de2b6b995a1bf527eb51f5c96fc5a/scripts/github-actions/get-changed-module.js#L1-L10

Reply:
Updated, thanks!